fedmsg.d/sse.py

```diff
@@ -0,0 +1,13 @@
+config = {
+
+    # SSE
+    "fmn.sse.pika.host": "localhost",
+    "fmn.sse.pika.port": 5672,
+    "fmn.sse.pika.exchange": "",
```

Comment:
It's not, I can actually remove it. Because the route /user/<userid>   and /group/<groupid>   the 'user'  and 'group'  are the name of the exchanges.